src/mcode-pilot/components/TreatmentOptionsOutcomes/TreatmentOptionsOutcomes.jsx

```diff
@@ -293,23 +208,30 @@ export default class TreatmentOptionsOutcomes extends Component {
                                         treatments={similarPatientTreatments}
                                         selectedTreatments={comparedTreatments}
                                         toggleTreatments={this.toggleTreatment('compared')}
-                                        treatmentNames={TREATMENT_NAMES}
                                     />
                                 }
                             </span>
                         </div>
 
-                        {comparedTreatmentCombinations.map((treatmentCombination, i) =>
-                            this.renderComparedRow(treatmentCombination, i, noIncludedRow ? null : includedRow)
+                        {comparedTreatmentData.map((treatmentData, i) =>
+                            this.renderComparedRow(treatmentData, i, noIncludedRow ? null : includedTreatmentData[0])
                         )}
                     </div>
                 </div>
-                <TableLegend includedRow={includedRow} treatmentNames={TREATMENT_NAMES}/>
+
+                {!noIncludedRow &&
+                    <TableLegend includedRow={includedTreatmentData[0]} />
```

Comment:
Without an included row the legend disappears but without any compared treatments the legend remains and is the only thing in the table.  Should the legend also disappear when `comparedTreatmentData` is empty?

Reply:
Technically, the first line of the legend applies to the included row, and it seems arbitrary to me to show only part of the legend, so I'd say we should leave it as is.